src/plugins/console/server/lib/proxy_request.ts

```diff
@@ -113,7 +113,10 @@ export const proxyRequest = ({
 
   const timeoutPromise = new Promise<any>((timeoutResolve, timeoutReject) => {
     setTimeout(() => {
-      if (!req.aborted && !req.socket) req.abort();
+      // Destroy the stream on timeout and close the connection.
+      if (!req.destroyed) {
+        req.destroy();
```

Comment:
According to the docs, [request.abort](https://nodejs.org/dist/latest-v16.x/docs/api/http.html#requestabort) method is deprecated and [request.destroy](https://nodejs.org/dist/latest-v16.x/docs/api/http.html#requestdestroyerror) should be used instead